core/browser_events.ts

```diff
@@ -95,24 +88,6 @@ export function conditionalBind(
   } else {
     node.addEventListener(name, wrapFunc, false);
     bindData.push([node, name, wrapFunc]);
-
-    // Add equivalent touch event.
-    if (name in Touch.TOUCH_MAP) {
-      const touchWrapFunc = (e: Event) => {
-        wrapFunc(e);
-        // Calling preventDefault stops the browser from scrolling/zooming the
-        // page.
-        const preventDef = !opt_noPreventDefault;
```

Comment:
Lint points out you're not using `opt_noPreventDefault`. Why is it no longer needed?